src/lib/stats/sampling.ml

```diff
@@ -61,27 +62,76 @@ let normal ?seed ~mean ~std () =
   let rsn = normal_std ?seed () in
   (fun () -> std *. (rsn ()) +. mean)
 
+(** Alias method for sampling from a categorical distribution.
+
+    The method allows to generate a sample in costant time after a
+    linear time preprocessing of the probabilities array. *)
+module Alias =
```

Comment:
I would also add a link to the source of the algorithm in this comment. Either Vose's paper or the Kevin Schwartz website would be great.